share/completions/emaint.fish

```diff
@@ -0,0 +1,32 @@
+## Global Opts
+complete -c emaint -s h -l help         -d "show this help message and exit"
+complete -c emaint -s c -l check        -d "Check for problems (a default option for most modules)"
+complete -c emaint      -l version      -d "show program's version number and exit"
+complete -c emaint -s f -l fix          -d "Attempt to fix problems (a default option for most modules)"
+complete -c emaint -s P -l purge        -d "Removes the list of previously failed merges. WARNING: Only use this option if you plan on manually fixing them or do not want them re-installed."
```

Comment:
The warning's a bit long. Maybe just "DANGEROUS"?

Reply:
Again, copied from emaint's usage output.
The flag isn't all that "dangerous" actually, it's more the help-message that is overly verbose.
Maybe just something along the lines of "see man-page"?

Reply:
Yeah, that'll work. Also use the imperative "Remove the list", not "Removes".